### DISCUSSION TITLE
Fastapi for building a trading strategy web app

### DISCUSSION THREAD

**Discussion by droidroy-ai** [fastapi/fastapi#2658] [Questions] [answered]
Hello guys, 

I have used fastapi for some simple ML model deployment. But currently I have a task of building a **full fledged trading strategy web app.** I was thinking of using Fastapi as the web framework for this task. Is Fastapi a good option for my entire task? Asking because I am becoming a bit confused on which web framework I should go for. 

Thank you.

**Comment by matthewchung74** [comment, marked as answer]
i have no idea but i did see this. https://www.youtube.com/playlist?list=PLvzuUVysUFOuoRna8KhschkVVUo2E2g6G .. might help.

**Reply by droidroy-ai** [reply]
Thank you. This actually gave me some ideas.
